src/index.html

```diff
@@ -1,12 +1,49 @@
 <!DOCTYPE html>
 <html lang="en">
+
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Calendar</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
+
   <body>
-    <h1>Calendar</h1>
+    <div class="wrapper">
+      <div class="calendar calendar--month-length--29 calendar--start-day--sun">
+        <div class="calendar__day calendar__day--day-1"></div>
```

Comment:
```suggestion
        <div class="calendar__day calendar__day--day--1"></div>
```
The value of a modifier is separated from its name by a double hyphen (--).

Reply:
God please save BEM)))